cmd/kube-controller-manager/app/certificates.go

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
Oh, right. That's the `app` as in `cmd/*/app`. Though you are splitting up the controllers as apps.